CHANGES.md

```diff
@@ -1,3 +1,30 @@
+## edge-19.12.2
+
+* CLI
+  * Added support for injecting CronJobs and ReplicaSets, and the ability to use
+    them as targets in the CLI subcomands
+  * Added ability to update the trust anchor and issuer certificates during
+    `linkerd upgrade` with the new flags `--identity-issuer-certificate-file`,
+    `--identity-issuer-key-file` and `identity-trust-anchors-file`
+
+* Controller
+  * Fixed bug causing pods having security context capabilities to fail to be
+    injected
+  * Upgraded Prometheus to v1.2.1 (thanks @daxmc99!)
```

Comment:
This is the Prometheus API (`client_golang`) package, not Prometheus itself. I think it can go under the _Internal_ section.

Reply:
Oops, good catch :+1: